corehq/apps/es/users.py

```diff
@@ -98,6 +98,12 @@ def unknown_users():
     """
     return filters.doc_type("UnknownUser")
 
+def system_users():
+    """
+    Returns system users. system users have a user_id of system
+    """
+    return username("system")
```

Comment:
might be good to use the constant for this: https://github.com/dimagi/commcare-hq/blob/9aa3a8ea8cd4bfd4782eb59b01b4cbb9b86c1e4a/corehq/apps/users/util.py#L22

Reply:
OOOOOH, i hadn't seen that. I'll also update the demo user one

Reply:
Although it's possible this module is trying to not reach into other ones too much

Reply:
^ if that's true then I would say having the string "system" is an invisible coupling which is harder to reason / know about. 
You could bump this out to `filters/users.py` instead which already imports `util` and do 

```py
user_type_filters.append(user_es.username(util.SYSTEM_USER_ID))
```

instead